posawesome/posawesome/api/payment_processing/processor.py

```
@@ -7,11 +7,116 @@
+            mpesa_doc = frappe.get_doc("Mpesa Payment Register", payment_name)
+            linked_customer = getattr(mpesa_doc, "customer", None)
+            payment_entry_name = getattr(mpesa_doc, "payment_entry", None)
+            if (
+                cint(getattr(mpesa_doc, "docstatus", 0)) == 1
+                and payment_entry_name
+                and (not linked_customer or linked_customer == customer)
+            ):
+                completed_entries.append(frappe.get_doc("Payment Entry", payment_entry_name))
+                continue
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)